Chore: lint + small edits (#596)
discord-userdoccers/discord-userdoccers: pages/resources/discovery.mdx

###### Guild Trait Structure

| Field          | Type       | Description                                     |
| -------------- | ---------- | ----------------------------------------------- |
| emoji_id       | ?snowflake | The ID of the emoji associated with the trait   |
| emoji_name ^1^ | ?string    | The name of the emoji associated with the trait |
| emoji_animated | boolean    | Whether the associated emoji is animated        |
| label          | string     | The name of the trait                           |

> | position       | integer    | The position of the trait in the array          |

Reviewer: not sure why any of these descriptions were changed... also the footnote should be on both emoji fields

> ^1^ Only standard Unicode emoji names are accepted; custom guild emoji are not supported and will result in a `GUILD_PROFILE_NO_CUSTOM_EMOJI` error

Reviewer: ```suggestion
^1^ Only standard Unicode emoji names are accepted; custom guild emoji are not supported and will result in an error.
```
specific form body errors are not documented
